### PR TITLE
Updated offer ID for veritas solution

### PR DESCRIPTION
**Pull request by swapnildombaleveritas**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
Please also change the details below:
1. The tier should be `Partner` not `Microsoft`
2. The support details also need to be changed; they should be partner details.

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
We are waiting for your updates on above asks. Thanks

**Comment by swapnildombaleveritas** [comment]
> Hi @swapnildombaleveritas, We are waiting for your updates on above asks. Thanks

Thanks for the review. Will make the changes and update the PR.

File: Solutions/Veritas NetBackup/SolutionMetadata.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
 {
 	"publisherId": "azuresentinel",
-	"offerId": "azure-sentinel-solution-netbackup",
+	"offerId": "veritas-netbackup-8-s",
```
Comment (v-atulyadav) [inline Solutions/Veritas NetBackup/SolutionMetadata.json:3]: Hi @swapnildombaleveritas,
OfferId must contain a sentinel word. Thanks

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
Please reply on above comment. Thanks

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
Please correct OfferId as we mentioned in above comment. Thanks

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
We are waiting for your reply on above asks. Thanks

File: Solutions/Veritas NetBackup/SolutionMetadata.json (hunk 1 above)
Comment (swapnildombaleveritas) [reply Solutions/Veritas NetBackup/SolutionMetadata.json:3]: Sure..will update

**Comment by swapnildombaleveritas** [comment]
This offer id "veritas-netbackup-8-s" we have generated for veritas for other offerings as well.
Kindly suggest if we need to generate new offer id with 'sentinel' name in it?

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
Please generate new offer with sentinel keyword. Thanks

**Comment by swapnildombaleveritas** [comment]
> Hi @swapnildombaleveritas, Please generate new offer with sentinel keyword. Thanks

Kindly help us to create new offer id. Should it be a new market place offer or do we have any new method for sentinel?
If we have any documentation which mention how to create new offering specifically for sentinel please let us know.

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
Please refer step 3 from below link. Thanks
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/README.md

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
We are waiting for your reply on above asks. Thanks

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
Please respond. Thanks

**Comment by v-atulyadav** [comment]
Hello @swapnildombaleveritas,
We are waiting for your reply. Thanks

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
Any update. Thanks

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
We have already shared the offer creation document with you, if you still have doubts then please let us know. Thanks

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
Can we connect over a call? Thanks

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
Could you please provide your email address so that we can arrange a call? Thanks

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
Please respond on above. Thanks

**Comment by v-atulyadav** [comment]
Hi @swapnildombaleveritas,
Please respond on above ask. Thanks

**Comment by v-atulyadav** [comment]
We wanted to check on the status of PR #10406. PR is pending for more than 60 days. Please let us know if you need any assistance to review this PR. Per our standard operating procedures if no response is received in the next 7 business days, we will close this PR. Thank you for your cooperation.

**Comment by v-atulyadav** [comment]
Since we have not received a response in the last 7 days, we are closing your PR #10406 per our standard operating procedures. If you still need support for this issue, you can re-open the PR at any time. 

If you do re-open, we simply request that you ensure the PR has response to the last request. Thank you for your cooperation.

**Pull request closed without merging**